bedrock/mozorg/templates/mozorg/credits-faq.html

```diff
@@ -13,7 +13,6 @@
 
 {% block canonical_urls %}
   <link rel="canonical" href="{{ canonical_url }}">
-  <link rel="alternate" hreflang="x-default" href="{{ canonical_url }}">
```

Comment:
There's also one defined in `bedrock/base/templates/404-locale.html` -- is that one ok?

Reply:
Yeah I believe so cos that page only gets seen by robots

Reply:
Sorry--can you clarify for me please? What is that page and how is it called? What is the URL that is indicated in the x-default tag for that page? 

If the URL doesn't return a 200 OK status code--if it redirects to another page or 404s--then it is not a valid value for the tag. 

As a reminder, one uses hreflang tags to indicate pages that you want to have indexed in search engines. If it's not intended to be seen by users, then it should not be included in hreflang tags.

Reply:
There are two pages in the thread here, so I'll cover them both:

1. The `credits-faq.html` template is used to render https://www.mozilla.org/credits/faq/ - which has no locale-specific version - it's always https://www.mozilla.org/credits/faq/ with a 200 OK.
This changeset removes the hreflang from /credits/faq/ - this is item 4 in the checklist in the main PR description.

2. `404-locale.html` is used to render the local-selection page that robots/spiders see if they go to https://www.mozilla.org -- as such, one can't see it in a browser, because that always sends something for the `Accept-language` header. It returns a 404 status code with the special content that shows all the available locales, IIRC.
This page currently has an x-default set - see https://github.com/mozilla/bedrock/blob/3bb94e5c8361e80443273016651ba840fbe0a451/bedrock/base/templates/404-locale.html#L31
This changset does NOT remove that. (This is item 2 in the checklist in the main PR description, though I can see how I didn't make it easy to join those dots)

Hope that is useful - LMK if i'm not giving you what you need @a-kyne

Reply:
1. Makes sense, thanks.

2. Sorry, I don't understand what you're saying about the 404 status code. What GET request would result in a robot getting a 404 server response for the locale page? 

I just visited https://www-demo8.allizom.org/, https://www.mozilla.org, and https://www.mozilla.org/ (using a browser profile that doesn't have a preferred language selected) and they all returned a 200 code with the locale-selection page. This is the expected result, not a 404 code.

Reply:
Both behave differently based on `is_root`, e.g.:
https://github.com/mozilla/bedrock/blob/f445fb61363d6cf84d906c5e0f31df6b4c0005fb/lib/l10n_utils/__init__.py#L103

and the same condition is used for `hreflang="x-default"` on it — so it's only being sent on the non-404 `is_root`, and is hidden on the 404 render anywhere else — so all should be OK.

Reply:
Thanks for the pointer @janbrasna! Much appreciated

Reply:
@a-kyne it sounds like I was wrong about the 404 code response (see Jan's comment above).

So if what you're getting is looking correct to you (and it sounds like it is) then I think we can merge this and see how things improve when we get it into production. Yeap?

Reply:
Sounds good to me! Thanks for your help on this.